tests/test_django_model_ref_field.py

```diff
@@ -51,6 +55,53 @@ class Config(ConfigurationModel):
     assert field.is_required() is True
 
 
+@pytest.mark.parametrize(
+    "invalid_values",
+    (
+        "",
+        "hello world",
+        "user@email.com",
+        "$price",
+        "my.variable",
+        "résumé",
+        "hello!",
+        "!",
+        "#",
+        "+",
+    ),
+)
+def test_slug_validation_fails_on_both_pydantic_and_django(invalid_values):
+
+    class Config(ConfigurationModel):
+        slug = DjangoModelRef(TestModel, "slug")
+
+    with pytest.raises(ValidationError):
+        Config(slug=invalid_values)
+
+    with pytest.raises(DjangoValidationError):
+        validate_slug(invalid_values)
+
+
+@pytest.mark.parametrize(
+    "valid_values",
+    (
+        "a",
```

Comment:
nitpicking, but I think the only character not tested now is `_`, which should be allowed by the regex